src/zepto.js

```diff
@@ -437,10 +436,12 @@ var Zepto = (function() {
     },
 
     ready: function(callback){
-      // need to check if document.body exists for IE as that browser reports
-      // document ready when it hasn't yet created the body element
-      if (readyRE.test(document.readyState) && document.body) callback($)
-      else document.addEventListener('DOMContentLoaded', function(){ callback($) }, false)
+      // don't use "interactive" on IE <= 10 (it can fired premature)
```

Comment:
Ah, sorry, I didn't realize you wrote `IE <= 10`. I misread that as `IE < 10`. Yes, we do want to support IE 10.